packages/frontend-api/src/Model/Product/Filter/ParameterFilterOption.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Product\Filter;
+
+use Shopsys\FrameworkBundle\Model\Product\Parameter\Parameter;
+
+class ParameterFilterOption
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Product\Parameter\Parameter
+     */
+    public Parameter $parameter;
```

Comment:
Are properties in *FilterOption classes `public` intentionally?

Reply:
Yes that are only "dumb" data classes to pass data to overblog bundle to transform them into graphql format